content/_snippets/string-number-formatting.md

```diff
@@ -3,5 +3,6 @@ XRP Ledger APIs generally use strings, rather than native JSON numbers, to repre
 * Base-10.
 * Non-zero-prefaced.
 * May contain `.` as a decimal point. For example, ½ is represented as `0.5`. (American style, not European)
-* May contain `E` or `e` to indicate being raised to a power of 10 (scientific notation). For example, `1.2E5` is equivalent to 1.2×10<sup>5</sup>, or `120000`.
+* Negative amounts start with the character `-`.
+* May contain `E` or `e` to indicate being raised to a power of 10 (scientific notation). For example, `1.2E5` is equivalent to 1.2×10<sup>5</sup>, or `120000`. Negative exponents are also possible.
 * No comma (`,`) characters are used.
```

Comment:
I wouldn't say we're expecting all users to accept the English-language format. We're using what's basically the native JavaScript format, but putting it in a string just as a precaution. So yeah, it uses `.` for a decimal point like the English-language convention, but it doesn't have thousands separators at all. I think most people writing code have come to terms with that, since it's not like the programming languages have different parsers based on what natural language you speak. (At least, I don't think they do!)

Reply:
I did a search and found this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
I think it's fair to continue doing what we have been doing for now, but consider if there is a need to manage the decimal separators in a more universal way in the future.

Reply:
The "locale string" formats are meant for humans to read, but they are sometimes ambiguous and are not suitable as a means of communicating numbers meant for machines to read.

The number format we're talking about here is primarily intended for machines to read and write. We should not add extra considerations like locale if we can avoid it. In any case, this section is describing how the `rippled` APIs behave, so any substantive changes to how the code actually behaves would be [`rippled`](https://github.com/ripple/rippled/) features, not doc changes here.